packages/metrics/src/create-gc-metrics/create-gc-metrics.ts

```diff
@@ -12,7 +12,7 @@ const defaultOptions = {
   labels: [],
   metricPrefix: '',
   metricNames: {
-    up: ['up'],
+    up: ['nodejs_up'],
```

Comment:
I suggest `nodejs_up` since it's related to the status of nodejs, and based on the context (the following metrics are all preffixed with `nodejs_`).

If anyone has better suggestion, please feel free to suggest another name, as long as it's different than `up`.